telemetry/event_counters.go

```diff
@@ -47,22 +58,33 @@ func EventSuccessCounter(
 // If err is not nil, the counter is not incremented but Prometheus will ingest this event.
 func ProofRequirementCounter(
 	reason prooftypes.ProofRequirementReason,
+	serviceId string,
+	applicationAddress string,
+	supplierOperatorAddress string,
 	err error,
 ) {
+	if !isTelemetyEnabled() {
+		return
+	}
+
 	incrementAmount := 1
-	isRequired := strconv.FormatBool(reason != prooftypes.ProofRequirementReason_NOT_REQUIRED)
 	labels := []metrics.Label{
-		{Name: "proof_required_reason", Value: reason.String()},
-		{Name: "is_required", Value: isRequired},
+		{Name: "reason", Value: reason.String()},
```

Comment:
```suggestion
		{Name: "proof_requirement_reason", Value: reason.String()},
```

Reply:
This is redundant in `poktroll_proof_requirements{reason=""}`